### PR TITLE
Fix: Sorting and Certifications

### PR DESCRIPTION
**Pull request by art29**

### CODE REVIEW

File: app/controllers/staff/training_sessions_controller.rb [hunk 1]
```diff
@@ -119,11 +119,13 @@ def certify_trainees
...
-        error = true
-        flash[
-          :alert
-        ] = "#{graduate.username}'s certification not saved properly!"
+      if certification.unique_cert
+        unless certification.save
+          error = true
+          flash[
+            :alert
+          ] = "#{graduate.username}'s certification not saved properly!"
```
Comment (NomadNaomie) [inline app/controllers/staff/training_sessions_controller.rb:127]: Tiny thing here, "did not save properly" is probably the way to word this

File: app/views/staff_dashboard/index.html.erb [hunk 2]
```diff
@@ -34,40 +34,32 @@
...
-              <span onclick="sortTable('staff-users-table', 2)"><%= fa_icon('arrow-down') %></span></th>
-            <th class="email-header"> # of visits <br> (Past 2
-              months)<span onclick="sortTable('staff-users-table', 2)"><%= fa_icon('arrow-down') %></span></th>
-            <th class="last-seen-header">Last Seen
-              <span onclick="sortTable('staff-recent-users-table', 2)"><%= fa_icon('arrow-down') %></span></th>
-            <th class="action-header">Sign Out of <%= @space.name.capitalize %></th>
+            <th class="certifications-header">Certifications in <%= @space.name %></th>
+            <th class="email-header"> # of visits <br> (Past 2 months)</th>
+            <th class="last-seen-header">Last Seen</th>
+            <th class="action-header disable-sort">Sign Out of <%= @space.name.capitalize %></th>
```
Comment (NomadNaomie) [inline app/views/staff_dashboard/index.html.erb:46]: I hate the look of .capitalize, I think all spaces currently have their correct capitalisation in .name

File: app/views/staff_dashboard/index.html.erb [hunk 3]
```diff
@@ -76,14 +68,12 @@
...
         <thead class="table-primary">
           <tr>
-            <th class="name-header">Name
-              <span onclick="sortTable('staff-recent-users-table', 0)"><%= fa_icon('arrow-down') %></span></th>
-            <th class="email-header">Email
-              <span onclick="sortTable('staff-recent-users-table', 1)"><%= fa_icon('arrow-down') %></span></th>
-            <th class="action-header">Sign In To <%= @space.name.capitalize %></span></th>
+            <th class="name-header">Name</th>
+            <th class="email-header">Email</th>
+            <th class="action-header disable-sort">Sign In To <%= @space.name.capitalize %></span></th>
```
Comment (NomadNaomie) [inline app/views/staff_dashboard/index.html.erb:76]: Same thing here

**Review by NomadNaomie — APPROVED** [review]

File: app/views/staff_dashboard/index.html.erb (hunk 2 above)
Comment (art29) [reply app/views/staff_dashboard/index.html.erb:46]: True

**Pull request merged**